source/core/NumericString.cpp

```diff
@@ -2715,6 +2807,8 @@ DEFINE_VIREO_BEGIN(NumericString)
     DEFINE_VIREO_FUNCTION(StringFormat, "p(i(VarArgCount) o(String)   i(String) io(ErrorCluster err) i(StaticTypeAndData))")
     DEFINE_VIREO_FUNCTION(StringScanValue, "p(i(String) o(String) i(String) o(StaticTypeAndData))")
     DEFINE_VIREO_FUNCTION(StringScan, "p(i(VarArgCount) i(String) o(String) i(String) i(UInt32) o(UInt32) io(ErrorCluster err) o(StaticTypeAndData))")
+    DEFINE_VIREO_FUNCTION(ByteArrayToString, "p(o(String) i(Array) i(Int32) io(ErrorCluster err))")
```

Comment:
For both functions how about inputs first and outputs last?

Reply:
I agree, that would be better.  I wish we were consistent everywhere, but it's generally the rule that the output is first when there are a variable number of inputs, and not otherwise.

Reply:
Will update. Looks like emission side is that way.